### DISCUSSION TITLE
POH Challenge Mode Active Indicator

### DISCUSSION THREAD

**Discussion by SitDownPro** [runelite/runelite#15669] [Ideas]
I am not sure where to ask or suggest this. Can we add a plug in to indicate if Challenge Mode is turned on in the POH? I use the flames in the basement to kill me for full hp. This stays active as long as you don't hop/log. Sometime I forget that I hopped and stand in flames like an idiot.

**Comment by robrichardson13** [comment]
Love this idea! Wrote a plugin that keeps track of this flag and shows text above the lever. It should work with all 3 levers. Also can configure the text and color.

If it gets approved/merged in, its called "Challenge Mode Indicator".

Code: https://github.com/robrichardson13/challenge-mode-indicator/tree/master
PR: https://github.com/runelite/plugin-hub/pull/3377

![2022-09-26 23 18 14](https://user-images.githubusercontent.com/9492530/192448046-107b15e1-4bea-4420-ab79-d282eb0a2be3.gif)

**Reply by robrichardson13** [reply]
@SitDownPro this is now live.

**Reply by SitDownPro** [reply]
![Screenshot 2022-10-06 060025](https://user-images.githubusercontent.com/113478343/194285090-b10812ec-bbaa-4965-aebb-e20c36e700da.png)

Bug shows in rouges den

**Reply by robrichardson13** [reply]
Moved conversation to: https://github.com/robrichardson13/challenge-mode-indicator/issues/1
